types/decimal.go

```diff
@@ -298,7 +302,11 @@ func (d Dec) String() string {
 		bzWDec[inputSize-10] = byte('.')
 		copy(bzWDec[inputSize-9:], bz[inputSize-10:])
 	}
-	return string(bzWDec)
+	if isNeg {
+		return "-" + string(bzWDec)
+	} else {
```

Comment:
I think this `else` can be replaced by a plain `return string(bzWDec)`

Reply:
was resolved